osnap/data/README.md

```diff
@@ -25,45 +25,58 @@ To import LTDB into OSLNAP, proceed with the following:
     - Click the button "Download Standard Data Files"
     - Repeat the process, this time selecting "sample" in the **select file
       type** menu and "All years" in the **select a year**
-- in your file browser, navigate to the two zip archives you just downloaded. By
-  default they are called `LTDB_Std_All_Sample.zip` and
-  `LTDB_Std_All_fullcount.zip`. Extract both of these zip archives into a single
-  directory containing all CSV files:
-
-[placeholder image]
+- Note the location of the two zip archives you just downloaded. By default they are called `LTDB_Std_All_Sample.zip` and
+  `LTDB_Std_All_fullcount.zip`. 
 
-Finally, to load the data into oslnap, call the `import_ltdb` function and pass the path of the directory containing the LTDB CSV files
+Finally, to load the data into osnap, call the `read_ltdb` function and pass the paths of the two zip archives you downloaded from the LTDB project page:
 
 ```
-from oslnap.data import import_ltdb
-import_ltdb("~/ltdb_data")
+from osnap.data import read_ltdb
+
+# if the archives were in my downloads folder, the paths might be something like this
+
+sample = "~/downloads/LTDB_Std_All_Sample.zip"
+full = "~/downlodas/LTDB_Std_All_fullcount.zip"
+
+read_ltdb(sample=sample, fullcount=full)
 
 ```
-    
+
+The reader will extract the necessary data from the archives, calculate some additional variables, and store the database as an apache parquet file. It will also return a pandas DataFrame if you want to get started right way or if you want inspect the variables.
+
+
+### Neighborhood Change Database
 
 
 
+## Creating Datasets for Analysis
 
-### Geolytics Neighborhood Change Database (NCDB)
+To perform neighborhood analyses, osnap provides the `Dataset` class which stores information about the spatial boundaries and social composition of a study area. 
 
-#### Datasets from Geolytics NCDB
+Creating a set of neighborhoods is as simple as instantiating the Dataset class with a location filter and a source database. The location filter can be either a `geopandas.GeoDataFrame` that defines the total extent of a boundary area (such as an MSA), or a list of state and county FIPS codes.
```

Comment:
Can we add arbitrary lists of indexes (say tracts that draw from neighboring MSAs)?

Reply:
thats a good idea. I'd intended the API to look like cenpy's `geo_filter`, so that argument could be passed directly in the case that users want 'census' or 'acs' as the source database

Reply:
Fwiw, if I were to write the cenpy api again, I'd just parse arbitrary keyword arguments into a Geo hierarchy, rather than making geo_unit and geo_filter a thing. So like, you pass `.query(..., state=x, county=y)` and it gives you the level below what's specified.